src/cli.ts

```diff
@@ -6,13 +6,14 @@ import {renderScreenshot} from "./index";
 
 const options = new Command("terminal-screenshot")
   .description("Render terminal ANSI output into images!")
-  .helpOption("-h, --help", "display usage help.")
-  .option("-d, --data [string]", "Data to be render to the terminal.")
-  .option("-m --margin [number]", "Margin to leave around the terminal area in pixels. (default: 0)")
-  .option("-f --font-family [string]", "Font family to use in terminal output. (default: Monaco)")
   .option("-b --background-color [css-color]", "Background color of the terminal. (default: black)")
+  .option("-c --color-scheme [string]        Color scheme of the terminal, i.e. theme. (default: '')")
```

Comment:
I suggest using `--xterm-theme` instead of `--color-scheme`, and accepting a path to a JSON file, containing the theme .. this should give users the maximum flexibility, without us having to explicitly override/configure this object?

Reply:
I agree using `theme` would be better, but we already have a `--type`/`-t` option, so using `--theme`/`-t` option is in conflict. I see several options here:
1. rename `--type` to ~`--format`/`-f`~ `--extension-type`/`-e` (we have `--font-family`/`-f`)
2. keep `--color-scheme`/`-c` <-- will stick to that for the time being

Reply:
Good catch. In that case, I think `-k --kind [png|jpeg]` would work better instead of `-e --extension-type`, as users might confuse that as appending an "extension" to the existing output file path.